src/Tasks/LinkableMigrationTask.php

```diff
@@ -0,0 +1,91 @@
+<?php declare(strict_types=1);
+
+namespace SilverStripe\LinkField\Tasks;
+
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\LinkField\Models\EmailLink;
+use SilverStripe\LinkField\Models\ExternalLink;
+use SilverStripe\LinkField\Models\FileLink;
+use SilverStripe\LinkField\Models\PhoneLink;
+use SilverStripe\LinkField\Models\SiteTreeLink;
+
+/**
+ * @deprecated 4.0.0 Will be removed without equivalent functionality.
+ */
+class LinkableMigrationTask extends BuildTask
+{
+    use MigrationTaskTrait;
+    use ModuleMigrationTaskTrait;
+
+    private static $segment = 'linkable-migration-task';
```

Comment:
```suggestion
    private static $segment = 'linkable-to-linkfield-migration-task';
```
Not a big deal but just more consistent with the other two.
You'll need to update any references to this, e.g. in the migration docs

Reply:
Updated